test/utilities/genetics-utils.js

```diff
@@ -27,16 +27,16 @@ describe("GeneticsUtils.convertDashAllelesToABAlleles()", function() {
   });
 
   it("should handle multiple alleles", function() {
-    const dashAlleles = "W-w,T-Tk,dl-d,Hl-hl",
+    const dashAlleles = "W-w,T-Tk,D-d,Hl-hl",
           abAlleles = GeneticsUtils.convertDashAllelesToABAlleles(dashAlleles);
-    assert.equal(abAlleles, "a:W,b:w,a:T,b:Tk,a:dl,b:d,a:Hl,b:hl", "should handle multiple alleles");
+    assert.equal(abAlleles, "a:W,b:w,a:T,b:Tk,a:D,b:d,a:Hl,b:hl", "should handle multiple alleles");
   });
 
-  it("should handle multiple alleles with extraneous spaces", function() {
-    const dashAlleles = " W - w , T - Tk , dl - d , Hl - hl ",
-          abAlleles = GeneticsUtils.convertDashAllelesToABAlleles(dashAlleles);
-    assert.equal(abAlleles, "a:W,b:w,a:T,b:Tk,a:dl,b:d,a:Hl,b:hl", "should handle multiple alleles with spaces");
-  });
+  // it("should handle multiple alleles with extraneous spaces", function() {
```

Comment:
Yes. Supporting the additional spaces just fell out of the original implementation as opposed to being an explicit goal. It's not supported by the schema validator, it didn't fall out of the new implementation with error detection, and it's probably a bad idea as it would decrease readability, so I figured it made sense to drop it. All of which seems like an argument for removing it entirely, but I guess I left it in just in case somewhere down the road someone might look at it and have a reason to revive it.